### PR TITLE
Add Customers resouce to Rubygem

### PR DESCRIPTION
**Pull request by chrismaddern**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+3.0.0 March 29, 2017
```
Comment (griffinmyers) [inline CHANGELOG.md]: I'd like to follow [semver](http://semver.org/) for this project.  Could we make this `v2.1.0` instead?  This introduces new functionality in a backwards-compatible way.

**Review by griffinmyers — CHANGES_REQUESTED** [review]
👏  This is awesome @chrismaddern!!  Thanks for contributing : )

Let's call this `2.1.0` instead.  When you make that change, we can merge.  I'll then cut a release on Github and publish to rubygems.

File: CHANGELOG.md (hunk 1 above)
Comment (chrismaddern) [reply CHANGELOG.md]: Good point @griffinmyers!

Will update now!

**Comment by chrismaddern** [comment]
@griffinmyers Ready for review! Thanks for handling deploy 👍

**Review by griffinmyers — APPROVED** [review]
👏 👏

**Pull request merged**